### PR TITLE
Return None on http code 204

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: apypie/api.py [hunk 1]
```diff
@@ -189,6 +189,8 @@ def http_call(self, http_method, path, params=None, headers=None, options=None,
 
         request = self._session.request(http_method, full_path, **kwargs)
         request.raise_for_status()
+        if request.status_code == 204:  # no content
+            return {}
```
Comment (ekohl) [inline apypie/api.py]: Why an empty dict instead of None? I think this is typically the response for a DELETE, but where else?

Comment (mdellweg) [reply apypie/api.py]: I have not tried None. I guess that is equally fine.

File: apypie/api.py [hunk 2]
```diff
@@ -189,6 +189,8 @@ def http_call(self, http_method, path, params=None, headers=None, options=None,
 
         request = self._session.request(http_method, full_path, **kwargs)
         request.raise_for_status()
+        if request.status_code == 204:  # no content
```
Comment (evgeni) [inline apypie/api.py]:
```suggestion
        if request.status_code == requests.codes.no_content:
```

File: apypie/api.py [hunk 3]
```diff
@@ -189,6 +189,8 @@ def http_call(self, http_method, path, params=None, headers=None, options=None,
...
+            return None
```
Comment (evgeni) [inline apypie/api.py]: this is not an empty dict ;) (and I know you changed that after opening the PR)
what was the reasoning behind the change?

I'm fine either way, just want to understand the difference

File: apypie/api.py (hunk 2 above)
Comment (evgeni) [reply apypie/api.py]: oh, and can we easily add a test for that?

File: apypie/api.py (hunk 3 above)
Comment (evgeni) [reply apypie/api.py]: oh, there was previous discussion on that. all good.

File: apypie/api.py (hunk 2 above)
Comment (mdellweg) [reply apypie/api.py]: Like this?

**Comment by evgeni** [comment]
travis has issues to fetch apt packages, but tests for 2.7 and 3.7 passed, good enough for me.

**Pull request merged**